src/main/java/emu/grasscutter/data/excels/MonsterData.java

```diff
@@ -1,25 +1,25 @@
 package emu.grasscutter.data.excels;
 
-import java.util.List;
-
 import emu.grasscutter.data.GameData;
 import emu.grasscutter.data.GameResource;
 import emu.grasscutter.data.ResourceType;
 import emu.grasscutter.data.ResourceType.LoadPriority;
 import emu.grasscutter.data.common.PropGrowCurve;
 import emu.grasscutter.game.props.MonsterType;
 
+import java.util.List;
+
 @ResourceType(name = "MonsterExcelConfigData.json", loadPriority = LoadPriority.LOW)
 public class MonsterData extends GameResource {
-	private int id;
-	
-	private String monsterName;
+    private int id;
+
+    private String monsterName;
     private MonsterType type;
     private String serverScript;
     private List<Integer> affix;
     private String ai;
     private int[] equips;
-    private List<HpDrops> hpDrops;
+    private List<emu.grasscutter.data.excels.MonsterData.HpDrops> hpDrops;
```

Comment:
Why?

Reply:
I will change it. It's my ide's problem.